worker-document/src/main/java/com/hpe/caf/worker/document/BulkDocumentMessageProcessor.java

```diff
@@ -118,6 +118,8 @@ public void processTasks() throws InterruptedException
                 // Set the response on the WorkerTask object
                 workerTask.setResponse(workerResponse);
 
+                // Unload the scripts
+                documentWorkerTask.unloadScripts();
```

Comment:
In `DocumentMessageProcessor` I note that you unload the scripts before you construct the response object so just for consistency with the non-bulk runtime I would move this up.